patches/minecraft/net/minecraft/util/ScreenShotHelper.java.patch

```diff
@@ -0,0 +1,14 @@
+--- ../src-base/minecraft/net/minecraft/util/ScreenShotHelper.java
++++ ../src-work/minecraft/net/minecraft/util/ScreenShotHelper.java
+@@ -52,7 +52,14 @@
+                 file2 = new File(file1, p_148259_1_);
+             }
+ 
++            net.minecraftforge.client.event.ScreenshotEvent.Pre preEvent = net.minecraftforge.client.ForgeHooksClient.onScreenshotPre(file2);
++            if (preEvent.isCanceled()) return preEvent.getCancelReason();
```

Comment:
You can make this 2 lines, and so much cleaner by moving code to the forge class.